### PR TITLE
Rebalances RCD construction costs around metal construction menu costs

### PR DESCRIPTION
**Pull request by freerealestate**
- Made costs and delays variables
- Made plasteel multiplier a variable
- Made shocked grilles shock unprotected RCD users
- Made grilles deconstruct instantly so the RCD isn't outdone by wirecutters
- Made the RCD unable to deconstruct reinforced floors (not that this really affects anything, just a neat consistency thing)
- Changed canRwall var to canRturf to indicate it covers both r walls and r floors
- Changed large matter cartridge matter amount to 160 so it can still fit in the RCD (I had previously set it to 200 not realising it wouldn't fit)
- Added fourth matter cartridge to RCD crate (one spawns in EVA) so it once again holds enough cartridges to fully load it

If you're wondering how I determined the cost values I used, the spreadsheet I used is [here](https://docs.google.com/spreadsheets/d/1ZvhRNn8KsnABcP-y2jEMV2BXq1Mqv2onKI5cQM_e9UE/edit?usp=sharing).

This pull was motivated by personal experience with the RCD being too powerful (e.g. able to build 53 walls per 50 sheets of metal inserted, as opposed to 10 by hand), and prompted by [this](https://tgstation13.org/phpBB/viewtopic.php?f=10&t=4360) feedback thread. I was a proponent of the pull to buff the RCD and wanted to be the one to fix the problems with it to make sure it was still a viable tool for construction.

### CODE REVIEW

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -294,7 +294,7 @@ RCD
 		if(1)
 			if(istype(A, /turf/space))
 				var/turf/space/S = A
-				if(useResource(1, user))
+				if(useResource(2, user))
```
Comment (ojacobson) [inline code/game/objects/items/weapons/RCD.dm]: Maybe make these values vars, so we can experiment with then using VV?

Comment (freerealestate) [inline code/game/objects/items/weapons/RCD.dm]: uh sure

i have no idea how to do that but i'll give it a go

**Comment by Core0verload** [comment]
RCD is useful in construction now. Leave it as is. :-1:

**Comment by Dorsisdwarf** [comment]
RCD is ridiculously *over* useful and a source of infinite materials, plus a super-effective combat tool.

Additionally this PR should solve the whole "Engineering RCDs are fully stocked, but the EVA one only has 1/3rd of a maximum charge" issue

**Comment by Jalleo** [comment]
I say this is suitable to prevent a exploit.

**Comment by Cebasiz** [comment]
can you adjust the energy draw on the cyborg RCD too

if the whole system is being buffed it probably should be adjusted up, while it's a useful module building anything expansive is a complete impossibility unless you have a 40k battery, something that almost never happens.

**Comment by Lumipharon** [comment]
RCD is still a super exploitable combat tool due to the sheer quantity of walls it can build or deconstruct (something like 50 walls or some shit?).

Make construction less tedious (ie: pointless time delays, extra construction steps like the screwdriver for deconning walls), rather then buffing one item, which leaves everyone without it still stuck with shit.

**Comment by freerealestate** [comment]
>RCD is still a super exploitable combat tool due to the sheer quantity of
walls it can build or deconstruct (something like 50 walls or some shit?).

Yeah, that's what this pull is addressing.

>Make construction less tedious

Beyond the scope of this pull.

**Comment by freerealestate** [comment]
Some feedback and/or opinions on this pull would be appreciated

It's been a week, I wouldn't mind hearing from maintainers if they agree with the change and if not what I've done wrong

**Comment by Miauw62** [comment]
this seems fine.

**Pull request merged**